### PR TITLE
Feat: upgrade service account

### PR DESCRIPTION
**Pull request by Kkhandale2024**
## what
* Improved service account

### CODE REVIEW

File: _example/varriables.tf
[the remark this answers is not in the document]
Comment (Kkhandale2024) [reply _example/varriables.tf:1]: done sir

**Review by AshutoshAM2002 — APPROVED** [review]
LGTM

File: _example/example.tf [hunk 1]
```diff
@@ -13,10 +22,11 @@ module "service-account" {
   label_order = var.label_order
 
   service_account_enabled     = true
-  project_id                  = "clouddrove"
+  project_id                  = "clouddrove-1"
   service_account_key_enabled = true
```
Comment (Tanveer143s) [inline _example/example.tf:26]: I think this resource also needed as bool
**google_service_account_iam_binding**  which mentioned in root main.tf

**Comment by Tanveer143s** [comment]
Add tflint workflow in .github/workflows

File: _example/example.tf (hunk 1 above)
Comment (Kkhandale2024) [reply _example/example.tf]: - done sir

**Review by VishwajitNagulkar — APPROVED** [review]

**Pull request merged**